app/src/androidTest/java/org/mozilla/focus/activity/URLCompletionTest.java

```diff
@@ -67,4 +91,206 @@ public void CompletionTest() throws InterruptedException, UiObjectNotFoundExcept
         TestHelper.hint.waitForExists(waitingTime);
         assertTrue (TestHelper.inlineAutocompleteEditText.getText().equals("http://www.mozilla.org"));
     }
+
+    @Test
+    public void CustomCompletionTest() throws InterruptedException, UiObjectNotFoundException {
+        // add custom urls in settings
+        // Enable custom autocomplete, and add 2 URLs
+        // Open Settings->Autocomplete menu
+        OpenCustomCompleteDialog();
+
+        // Enable URL autocomplete, and tap Custom URL add button
+        toggleAutocomplete();
+        addAutoComplete("680news.com");
```

Comment:
\[PMD\] INFO: The String literal "680news\.com" appears 4 times in this file; the first occurrence is on line 104